app/master/build.py

```diff
@@ -81,7 +83,6 @@ def api_representation(self):
         return {
             'id': self._build_id,
             'status': build_state,
-            'artifacts': self._artifacts_archive_file,  # todo: this should probably be a url, not a file path
```

Comment:
I removed this because it was an API field whose value was a local path. I think originally we might have planned to make it a url, but I'd rather do something like that as part of a larger effort to make the API more discoverable instead of only this one-off.